index.html

```diff
@@ -89,7 +89,7 @@ <h1>Tonic</h1>
               <li>CSS</li>
               <li>javaScript</li>
             </ul>
-            <span>See Project</span>
+            <span class="see-more-action" id="project1">See Project</span>
           </div>
         </div>
       </section>
```

Comment:
- [Optional] Please make sure that the project section is loaded dynamically from the js file. You should not have any project here, instead, your project array in your js file should be used to populate this project section. This is part of the project requirements. Kindly fix this.